index.d.ts

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
Can you please explain why this format is needed over the current format? Is this something that's specific to TypeScript?

Reply:
This is specific to TypeScript and ES6 - https://github.com/Microsoft/TypeScript/issues/3337#issuecomment-107971371. The `import React from "react";` works with babel and other transpilers. It requires having specific settings in tsconfig, which may not be set. These settings are not set as default. When these settings are not set, TypeScript will fail.

Reply:
Thank you for accepting my PR @MatthewHerbst!